docs/data-sources/nlb_service_list.md

```diff
@@ -2,9 +2,8 @@
 page_title: "exoscale_nlb_service_list Data Source - terraform-provider-exoscale"
 subcategory: ""
 description: |-
-Fetch Exoscale [Network Load Balancers (NLB)](https://community.exoscale.com/documentation/compute/network-load-balancer/) Services.
-
-Corresponding resource: [exoscale_nlb](../resources/nlb.md).
+  Fetch Exoscale Network Load Balancers (NLB) https://community.exoscale.com/documentation/compute/network-load-balancer/ Services.
+  Corresponding resource: exoscale_nlb ../resources/nlb.md.
```

Comment:
```suggestion
  Corresponding resource: [exoscale_nlb](../resources/nlb.md).
```

No?

Reply:
This is a side-effect of the `plainmarkdown` template function.
We run our descriptions through it in all other templates.

For example in the `exoscale_block_storage_volume_snapshot` data source, the corresponding resource has had its markdown formatting removed in the frontmatter
https://github.com/exoscale/terraform-provider-exoscale/blob/master/docs/data-sources/block_storage_volume_snapshot.md?plain=1

This should not cause any rendering issues.

Reply:
indeed, it's doing nothing :-)

```html
    <meta name="description" content="">
```

https://registry.terraform.io/providers/exoscale/exoscale/latest/docs/data-sources/block_storage_volume_snapshot